components/general/LatestNews.tsx

```diff
@@ -0,0 +1,40 @@
+import { titleFont } from "@/pages/_app";
+import styles from "@/styles/LatestNews.module.css";
+import Card from "../base/Card";
+import Button from "../base/Button";
+
+const LatestNews = () => {
+  return (
+    <section className={styles.latest_news}>
+      <div className={styles.latest_news__header}>
+        <h1
+          className={`${titleFont.className} global__uppercase-heading--small ${styles.latest_news__title}`}
+        >
+          LATEST NEWS
+        </h1>
+        <Button variant="secondary">SEE ALL</Button>
+      </div>
+      <article className={styles.latest_news__article}>
+        <Card
+          image={"/images/two_people_praying.png"}
+          cornerMakerLeft={"Branding, Design"}
+          title={"Things to Look for When Comparing Branding Alternatives"}
+        />
+        <div className={styles.latest_news__divider} />
```

Comment:
Extract it into a divider component? Or do you think a div is okay for now?

Reply:
Do you mean styled component?